server/routes/induction/common/additionalTrainingController.ts

```diff
@@ -43,7 +44,9 @@ export default abstract class AdditionalTrainingController extends InductionCont
       previousTraining: {
         ...inductionDto.previousTraining,
         trainingTypes: additionalTrainingForm.additionalTraining,
-        trainingTypeOther: additionalTrainingForm.additionalTrainingOther,
+        trainingTypeOther: additionalTrainingForm.additionalTraining.includes(AdditionalTrainingValue.OTHER)
```

Comment:
Only map the "other" text field if the corresponding checkbox values includes "OTHER"
We did this in most places in the induction journey, but there are a few places like this and the ones that follow where we hadn't